src/main/java/com/google/cloud/discotoproto3converter/disco/Document.java

```diff
@@ -127,6 +127,10 @@ public static Document from(DiscoveryNode root) {
     return thisDocument;
   }
 
+  // This method returns the same as parseMethods(), but returns all methods in a map indexed by
+  // method name.
+  //
+  // TODO: Combine with parseMethods?
```

Comment:
```suggestion
  // TODO: See <REPLACE ME with link to new bug filed in https://github.com/googleapis/disco-to-proto3-converter/issues>
```

Same applies to the 2nd TODO comment in this PR

Reply:
Done.